### PR TITLE
[🐴] add link to chat settings from main settings

### PR DESCRIPTION
**Pull request by mozzius**
Fixes https://github.com/bluesky-social/social-app/issues/4195

Added under basic

image

### CODE REVIEW

File: src/screens/Messages/Settings.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export function MessagesSettingsScreen({}: Props) {
 
   return (
     <CenteredView sideBorders style={a.h_full_vh}>
-      <ViewHeader title={_(msg`Settings`)} showOnDesktop showBorder />
+      <ViewHeader title={_(msg`Chat Settings`)} showOnDesktop showBorder />
```
Comment (haileyok) [inline src/screens/Messages/Settings.tsx:59]:
```suggestion
      <ViewHeader title={_(msg`Chat Preferences`)} showOnDesktop showBorder />
```

**Review by haileyok — APPROVED** [review]

File: src/view/screens/Settings/index.tsx [hunk 2]
```diff
@@ -647,6 +647,31 @@ export function SettingsScreen({}: Props) {
...
+          accessibilityLabel={_(msg`Chat settings`)}
+          accessibilityHint={_(msg`Opens chat settings`)}>
+          <View style={[styles.iconContainer, pal.btn]}>
+            <FontAwesomeIcon
+              icon={['far', 'comment-dots']}
+              style={pal.text as FontAwesomeIconStyle}
+            />
+          </View>
+          <Text type="lg" style={pal.text}>
+            <Trans>Chat Settings</Trans>
```
Comment (haileyok) [inline src/view/screens/Settings/index.tsx:672]:
```suggestion
            <Trans>Chat Preferences</Trans>
```

We use "Preferences" everywhere else here, but fwiw we _are_ using "Settings" on the actual settings screen itself...

Comment (haileyok) [reply src/view/screens/Settings/index.tsx:672]: Don't feel strongly, we can change this whenever.

Comment (mozzius) [reply src/view/screens/Settings/index.tsx]: I think Settings here, just so it's the same when you click through. Don't feel strongly either tho

Comment (haileyok) [reply src/view/screens/Settings/index.tsx]: Yea I realized that after going into the screen. Another option might be to just drop the "Settings" all together. You're in "Settings" already so you know it takes you to more settings.

Again though, that doesn't follow the other behavior we have lol...probably just something to revisit when we rework this screen hopefully soon.

Comment (mozzius) [reply src/view/screens/Settings/index.tsx]: can't wait to burn it all down fr

**Pull request merged**